.npmignore

```diff
@@ -0,0 +1,6 @@
+.gitignore
+.travis.yml
+.editorconfig
+gulpFile.js
+test.js
+screenshot.gif
```

Comment:
Use the `files` property in package.json instead. It's better to whitelist files than blacklist.